Remove stale reference to pngmath

jarrodmillman opened the pull request:
Completes fix for #2340 (see commit ffa3e5deb837).

This could be merged as is, but I noticed more cruft in the `conf.py` that I am happy to fix and include in the PR.  For example, there are still references to the old Trac site.   I am not planning to change anything substantial and I suspect nothing I do will be user visible.

Should I include additional Sphinx clean up commits in this PR or would you prefer I create a new PR for that purpose?  Either way, I can commit the changes over the weekend.

dschult commented:
Go ahead and put the changes into this PR.
Let's see if we can get conf.py cleaned up.

jarrodmillman commented:
This is ready to merge.  I rebased on master and tried to make my commit messages self-explanatory.  I was able to build the documentation on my system in a virtualenv.  I fixed one import warning, but this mostly removes old cruft.
